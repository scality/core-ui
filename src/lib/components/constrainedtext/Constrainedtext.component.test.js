import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/constrainedtext/__snapshots__"
});
