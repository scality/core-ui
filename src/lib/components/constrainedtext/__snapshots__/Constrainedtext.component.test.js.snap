// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Constrainedtext Default 1`] = `
<div
  className="sc-ifAKCX ugGGx"
>
  <h3
    className="sc-EHOje ftwWfK"
  >
    Constrained Text
  </h3>
  <div
    style={
      Object {
        "color": "#0F7FFF",
        "width": "100px",
      }
    }
  >
    <div
      className="sc-bxivhb fJlOgN"
    >
      <div
        className="sc-bdVaJa jCeSos sc-tooltip"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          className="sc-htpNat hRShHO sc-constrainedtext"
        >
          This is a long phrase
        </div>
      </div>
    </div>
  </div>
  <h3
    className="sc-EHOje ftwWfK"
  >
    With the ability to customize the style of tooltip
  </h3>
  <div
    style={
      Object {
        "color": "#0F7FFF",
        "width": "100px",
      }
    }
  >
    <div
      className="sc-bxivhb fJlOgN"
    >
      <div
        className="sc-bdVaJa jCeSos sc-tooltip"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          className="sc-htpNat hRShHO sc-constrainedtext"
        >
          This is a long phrase
        </div>
      </div>
    </div>
  </div>
</div>
`;
